indico/modules/events/controllers/admin.py

```diff
@@ -184,3 +185,16 @@ class RHAutoLinkerConfig(RHAdminBase):
     })
     def _process(self, rules):
         autolinker_settings.set('rules', rules)
+
+
+class RHDataRetentionSettings(RHAdminBase):
+    """Manage minimun and maximum data retention period in the admin area."""
+
+    def _process(self):
+        form = DataRetentionSettingsForm(obj=FormDefaults(**data_retention_settings.get_all()))
+        if form.validate_on_submit():
+            data_retention_settings.set_multi(form.data)
+            flash(_('Settings have been saved'), 'success')
+            return redirect(url_for('events.data_retention'))
+        return WPEventAdmin.render_template('admin/data_retention.html',
+                                            'data_retention', form=form)
```

Comment:
```suggestion
        return WPEventAdmin.render_template('admin/data_retention.html', 'data_retention', form=form)
```

I think that one fits in one line